cli/internal/config/config.go

```diff
@@ -13,7 +13,11 @@ import (
 	"github.com/tidwall/sjson"
 )
 
-var log = logger.New("config")
+var (
+	log                   = logger.New("config")
+	TelemetryWriteKey     = ""
```

Comment:
Let's add comment on top of this to let everyone know that is going to be populated through ldflags, in case someone moves that and forgets to change the paths.

Reply:
Will do !